client/src/components/EventForm.js

```diff
@@ -0,0 +1,190 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/macro';
+import isValidEvent from '../lib/validateEventForm';
+import DefaultButton from '../components/DefaultButton';
+
+
+export default function EventForm({ open }) {
+    const createEvent = {
+        event_title: '',
+        event_location: '',
+        event_month: '',
+        event_day: '',
+        event_description: '',
+    }
+    const [newEvent, setNewEvent] = useState(createEvent)
+    const [valid, setValid] = useState(false)
+
+
+
+
+    const handleChange = event => {
+        const field = event.target;
+        const value = field.value;
+
+        setNewEvent({
+            ...newEvent,
+            [field.name]: value,
+        });
+        console.log(newEvent)
+    };
+
+    function submitForm(event) {
+        event.preventDefault();
+        if (isValidEvent(newEvent)) {
+            setValid(true);
+            localStorage.setItem('newEvent', JSON.stringify(newEvent));
+
+        }
+    }
+
+    return (
+        <>
```

Comment:
I think you don't need those Fragments as everything is wrapped in a single element